### ISSUE TITLE
Show Council Districts layer on map

### ISSUE THREAD

**Issue by TylerMatteo** [NYCPlanning/ae-cp-map#12]
# Description
User sees Council District outlines on map when City Council District filter type is selected in Filter Menu.

## Acceptance Criteria

- [ ] When user selects "City Council Districts" from District Type dropdown in `FilterMenu` component, user sees CCD outlines and labels on the map
- [ ] District boundary styles match [specs in Figma](https://www.figma.com/design/LYHHoPop9l0jpEivk5CFzJ/Capital-Projects-Portal?node-id=670-5575&t=bHnqp3D5d54Ghn2t-0)

## Out of scope
* Additional map styling for _selected_ CCD - to be handled in separate Issue for map filtering

## Related work
* Blocked by https://github.com/NYCPlanning/ae-zoning-api/issues/254

**Comment by TangoYankee** [comment]
blocked by #18

**Comment by TylerMatteo** [comment]
This should be unblocked now that the tiles endpoint is implemented and we have the changes in PR #27 merged into the front end. The changes in that PR hook up the `<FilterMenu>` component to the router such that the selected type is tracked by the `districtType` query parameter. Implementation for this Issue can use that query param to control the `visible` prop on a new `MVTLayer` for CCDs.
